src/Mandarin.ViewModels/MandarinViewModelsServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Mandarin.ViewModels.Index;
+using Mandarin.ViewModels.Index.Carousel;
+using Mandarin.ViewModels.Index.OpeningTimes;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Mandarin.ViewModels
+{
+    public static class MandarinViewModelsServiceCollectionExtensions
+    {
+        public static void AddMandarinViewModels(this IServiceCollection services)
+        {
+            services.AddSingleton<IIndexPageViewModel, IndexPageViewModel>();
+            services.AddSingleton<ICarouselViewModel, CarouselViewModel>();
+            services.AddSingleton<IOpeningTimesViewModel, OpeningTimesViewModel>();
```

Comment:
ViewModels should probably be scoped.
Underlying models should be singleton if they are a single source of truth.

Reply:
When data is coming from another source, will do this.